css/src/components/persona.scss

```diff
@@ -0,0 +1,82 @@
+$persona-avatar-dimensions: 16 24 32 36 48 !default;
+$persona-avatar-background-color: $alternate-background-medium !default;
+$persona-avatar-font-color: $text-invert !default;
+$persona-default-size: 32px !default;
+$persona-font-size: $font-size-7 !default;
+$persona-gap-size: $layout-1 !default;
+$persona-metadata-font-size: $font-size-9 !default;
+$persona-metadata-font-color: $text-subtle !default;
+$persona-dimension-font-size-ratio: 0.45;
```

Comment:
Should that component just use `em` sizing instead of a calculated font-size?

Reply:
I thought calculating was a good idea cause it would help keep the ratio perfectly the same for all sizes, but I can make another set of variables for font sizes

Reply:
If the ratios aren't kept, then we're probably missing a spot in a subcomponent where we should be using `em`.

Reply:
Refactored this, let me know what you think.